src/main/java/uk/gov/hmcts/reform/ccd/document/am/Application.java

```diff
@@ -21,7 +21,7 @@
 public class Application {
 
     public static void main(final String[] args) {
-        SpringApplication.run(Application.class, args);
+        SpringApplication.run(Application.class);
```

Comment:
If We need to pass java command line parameter in future.How will we pass to it as you remove args parameter?

Reply:
Sonar does not allow unsafe command line parms and we should make sure that command line arguments are used safely here. e.g. we should sanitise it manually and either fix it if it’s a real vulnerability or resolve it as reviewed if it’s safe.